Utilities/WDLs/CreateIGVSession.wdl

```diff
@@ -0,0 +1,141 @@
+version 1.0
+
+workflow CreateIGVSession {
+    input {
+        Array[String]? bams
+        Array[String]? vcfs
+        Array[String]? interval_files
+
+        String ref_fasta
+
+        String output_name = "igv_session"
+    }
+
+    call MakeIGVXML {
+        input:
+            bams=bams,
+            vcfs=vcfs,
+            interval_files=interval_files,
+            ref_fasta=ref_fasta,
+            output_name=output_name
+    }
+
+
+    output {
+        File igv_session = MakeIGVXML.igv_session
+    }
+}
+
+task MakeIGVXML {
+    input {
+        Array[String]? bams
+        Array[String]? vcfs
+        Array[String]? interval_files
+
+        String ref_fasta
+
+        String output_name = "igv_session"
+    }
+
+    Int disk_size = 50
+    Int cpu = 2
+    Int memory = 8
+
+    command <<<
+        set -xueo pipefail
+
+        python << CODE
+        import xml.etree.ElementTree as ET
+
+        # Parse WDL array into Python lists
+        bams_list = ["~{default="" sep="\" , \"" bams}"]
+        vcfs_list = ["~{default="" sep="\" , \"" vcfs}"]
+        interval_files_list = ["~{default="" sep="\" , \"" interval_files}"]
+
+        # Build XML file using nodes in ElementTree, with 'Session' at root
+        session = ET.Element('Session')
+
+        # Check reference sequence based on Terra defaults; else use provided reference
+        if "~{ref_fasta}" == "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.fasta":
+            session.set('genome', 'hg38')
```

Comment:
Yeah that's a good point. I wasn't sure if it was even worth distinguishing between that case, since it's technically all handled by the last case if the user inputs a full ref fasta file. I changed it to just take in a `reference` string, which can either be a fasta path, or could be the built-in `hg38` or `hg19`, and is detected automatically from the standard Terra references. But I'm actually not sure if there's any benefit to splitting this way, since IGV might stream its references from the cloud even in those built-in cases...